src/System.Net.Sockets/tests/FunctionalTests/DualModeSocketTest.cs

```diff
@@ -620,8 +620,7 @@ public void DualModeConnectAsync_Static_DnsEndPointToHost_Helper(IPAddress liste
     [Trait("IPv6", "true")]
     public class DualModeBind : DualModeBase
     {
-        [Fact] // Base case
-        // "The system detected an invalid pointer address in attempting to use a pointer argument in a call"
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsSubsystemForLinux))] // https://github.com/Microsoft/BashOnWindows/issues/982
```

Comment:
Given how many of these conditionals we have, and given that we're likely to have at least some of them for some time, I wonder if we should add an entry to the platforms enumeration in buildtools and use ActiveIssue instead?